litellm/proxy/proxy_server.py

```diff
@@ -39,6 +40,10 @@
     import backoff
     import yaml
 
+    warnings.warn(
+        "Installed runtime dependencies for proxy server. Specify these dependencies explicitly with `pip install litellm[proxy]`"
```

Comment:
can we add this somewhere on docs too ?
perhaps here https://docs.litellm.ai/docs/simple_proxy#quick-start

Reply:
https://github.com/BerriAI/litellm/pull/966/commits/6fb178dc942c1d1844c192e472af7105c5012384